kong/plugins/acme/migrations/003_350_to_360.lua

```diff
@@ -10,11 +10,11 @@ return {
                 '{storage_config,redis}',
                 config #> '{storage_config, redis}'
                 || jsonb_build_object(
-                  'password', config #> '{storage_config, redis, auth}',
-                  'server_name', config #> '{storage_config, redis, ssl_server_name}',
+                  'password', COALESCE(config #> '{storage_config, redis, auth}', config #> '{storage_config, redis, password}'),
```

Comment:
Should this be otherway around? First try new and then old? Or is it?

Reply:
Hm... I'm not sure the order matters here actually 🤔 I mean technically it does but in this case there could be one of the 2 scenarios, right?
1. The data exists in: `storage_config.redis.auth` meaning we're at the 1st step of migration - just running `kong migrations up` and we haven't `kong migrations finish` yet. In such case it's safe to pull old data as during the upgrade process there could be no changes to admin api.
2. There's no data in `storage_config.redis.auth` so either there never was or we are rerunning this migration after someone already ran `kong migrations finish` in such case `storage_config.redis.auth` will be nil and we'll try to read data from `storage_config.redis.password`

Reply:
Yes, probably doesn't matter. Just thinking that safer to live new value alone if it is specified. Up to you.